gamersnet_backend/persistence/posts.js

```diff
@@ -54,5 +55,16 @@ async function addPost(userID, description, gameName, numPlayers, gameTimeUTC, d
   })
 }
 
+async function deletePost(_id, userID){
+  let db = await MongoDB.open();
+
+  let posts = db.collection('posts');
+
+  // just making sure the post is only deleted by the user that created it , just a note; for some reason (unknown)
+  // passing in userID was not working 
```

Comment:
where did you want to pass in user id? the query?

Reply:
UserID is getting passed but for some reason(not important now) it was not working;

The new commit will only allow users that created the post to delete them.